doc/README.md

```diff
@@ -25,8 +25,7 @@ It is brought to you by [Erlang Solutions](https://www.erlang-solutions.com/) an
 
 For a quick start just download:
 
-* The [pre-built packages](https://www.erlang-solutions.com/resources/download.html)
-that suit your platform (Ubuntu, Debian, CentOS, and macOS)
+* The [pre-built packages](https://www.erlang-solutions.com/resources/download.html) that suit your platform (Ubuntu, Debian, CentOS, and macOS)
```

Comment:
yeah, I changed them both to "suit" as in "the packages suit". I think I commmited the other thing to master:/ sorry